src/Workspaces/TestAnalyzerReference/Microsoft.CodeAnalysis.TestAnalyzerReference.csproj

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We had a test project for [Source generator](https://github.com/dotnet/roslyn/tree/main/src/Workspaces/TestSourceGenerator), that was used in integration tests: https://github.com/dotnet/roslyn/blob/19284be0905408367a4a953f8ff86a3715bebeab/src/VisualStudio/IntegrationTest/New.IntegrationTests/Microsoft.VisualStudio.LanguageServices.New.IntegrationTests.csproj#L50

I followed a similar approach and added one for refactorings/fixes/analyzers. I can as well merge this with the existing `TestSourceGenerator` project and just rename it. I wasn't sure if that would break existing tests, but let me give that a shot.

Reply:
Changed with https://github.com/dotnet/roslyn/pull/64364/commits/f406fff66a0e0d562c68daf4c6fdcdd673d1a4c2